pyctcdecode/decoder.py

```diff
@@ -688,8 +689,6 @@ def parse_directory_contents(filepath: str) -> Dict[str, Union[str, None]]:
         contents = os.listdir(filepath)
         # filter out hidden files
         contents = [c for c in contents if not c.startswith(".") and not c.startswith("__")]
-        if len(contents) not in {1, 2}:  # always alphabet, sometimes language model
```

Comment:
Can we relax this error here?

It is already checked that:

- the `"alphabet.json"` file is present
- `"language_model"` directory is present which is then further down the road checked aggressively

IMO there is no "danger" in having more than those files in the directory that is to be loaded. E.g. on `huggingface.co` we always have a `README.md` in the folder as well - see: https://huggingface.co/kensho/beamsearch_decoder_dummy/tree/main

Would it be ok for you to remove those lines here? @mikeyshulman

Reply:
im ok with it. Do the remaining tests pass? I think we might need to change e.g. 
https://github.com/kensho-technologies/pyctcdecode/blob/main/pyctcdecode/tests/test_decoder.py#L546

Reply:
I'll update the tests